frontend/src/__tests__/cypress/cypress/tests/e2e/settings/clusterSettings/testAdminClusterSettings.cy.ts

```diff
@@ -52,7 +52,7 @@ describe('Verify that only the Cluster Admin can access Cluster Settings', () =>
   });
   it('Test User - should not have access rights to view the Cluster Settings tab', () => {
     cy.step('Log into the application');
-    cy.visitWithLogin('/', TEST_USER);
+    cy.visitWithLogin('/', CONTRIBUTOR_USER);
```

Comment:
Can we rename this to ldap_contributor_user? Or something more descriptive.

Reply:
Fixed in [498164e](https://github.com/opendatahub-io/odh-dashboard/pull/3469/commits/498164e80896f458e9b029d87a341a52ddc1fddf)